getting_hired/README.md

```diff
@@ -1,20 +1,28 @@
-Web development is a lifelong journey of learning and growth. Continue that journey on a professional development team! You'll learn where to find jobs, how to do great interviews, and the best strategies to launch your career.
+# Getting Hired
 
-## The Outline
+This folder contains lesson markdown files that make up the <Getting Hired> course. This course exists in the [Full Stack Ruby On Rails](https://www.theodinproject.com/paths/full-stack-ruby-on-rails) and [Full Stack JavaScript](https://www.theodinproject.com/paths/full-stack-javascript) path on the Odin Project Website.
```

Comment:
These angle bracket needs to be removed, otherwise "Getting Hired" does not show up in the README preview:

```suggestion
This folder contains lesson markdown files that make up the Getting Hired course. This course exists in the [Full Stack Ruby On Rails](https://www.theodinproject.com/paths/full-stack-ruby-on-rails) and [Full Stack JavaScript](https://www.theodinproject.com/paths/full-stack-javascript) path on the Odin Project Website.
```

Reply:
done